kubevirt-disk-uploader-task.yaml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubevirt-disk-uploader-tekton
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
you can drop this role, because cluster role edit already contains all of these resources with required verbs:
```
oc get clusterrole edit -o yaml | grep -Ei "secret|virtualmachines|virtualmachineexports"
  - secrets
  - virtualmachines
  - virtualmachineexports
(output filtered)
```

Reply:
Thanks!